src/Ot2Rec/main.py

```diff
@@ -683,7 +683,14 @@ def create_align_yaml_stacked():
     update_align_yaml_stacked()
 
 
-def run_align():
+def create_stacks():
+    """
+    Method to only create stacks using IMOD but omit alignment
```

Comment:
Only one tiny comment: could we put in the docstrings what this function returns?

Reply:
Sorry, I meant the info on what create_stacks does, getting the stack image and the raw tilt angles?